src/app/demo/demo.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, OnInit } from '@angular/core';
+import {
+  NgbModal,
+  NgbModalRef
+} from '@ng-bootstrap/ng-bootstrap';
+import { constants } from '../constants';
+
+
+@Component({
+  selector: 'app-demo',
+  templateUrl: './demo.component.html',
+  styleUrls: ['./demo.component.scss']
+})
+export class DemoComponent implements OnInit {
+  checkboxStatus: boolean;
```

Comment:
Is there any use of this property? Remove if not using, Check for all the properties, methods, imports in all the files